### ISSUE TITLE
Icemining Errorr

### ISSUE THREAD

**Issue by ericb2233** [RainbowMiner/RainbowMiner#2482]
Getting the following error while mining Icemining:

WARNING: Pool API (Icemining) returned nothing.

**Comment by whitindllc** [comment]
its been broke for months.... the ssl key or api dont play well with RBM

**Comment by ericb2233** [comment]
Ok. So does mining even work with that pool?

**Comment by whitindllc** [comment]
not with RBM.....

**Comment by whitindllc** [comment]
> Ok. So does mining even work with that pool?

yes the pool is fine just does not work with RBM... works fine if you mine manually
